app/controllers/application_controller.rb

```diff
@@ -1,4 +1,14 @@
 class ApplicationController < ActionController::Base
+  include Pundit
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
   protect_from_forgery with: :exception
   before_action :authenticate_user!
+  private
+
+ def user_not_authorized(exception)
+   policy_name = exception.policy.class.to_s.underscore
```

Comment:
I think we can get rid of this line now that we're not interpolating variables into the flash message.